lms/views/api/lti.py

```diff
@@ -96,22 +96,23 @@ class CanvasPreRecordHook:
     def __init__(self, request):
         self.request = request
 
-    def __call__(self, score=None, request_body=None):
-        request_body["resultRecord"].setdefault("result", {})["resultData"] = {
-            "ltiLaunchUrl": self.get_speedgrader_launch_url()
-        }
+    def __call__(self, score=None, request_body=None) -> dict:
```

Comment:
This whole file seem to be about grading. I think it could be renamed. 

That been said I'd like to tackle that in a separate PR to avoid some rebase-hell.